actionview/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Rename `ActionView::Helpers::TagHelper#class_names` to `token_list` and
+    ensure the resulting Token list items are unique.
+
+        token_list("foo", "foo bar")
+        # => "foo bar"
+
+    *Sean Doyle*
```

Comment:
Since `class_names` hasn't been released yet, I think it might do to update the [existing CHANGELOG entry](https://github.com/rails/rails/blame/7af59e16a255da5aee2863af0eefaa9f0cfd857d/actionview/CHANGELOG.md#L46-L48) instead.  Or to remove the existing entry in favor of yours.  Either way, make sure you and everyone else gets credit! :smiley:

Reply:
@jonathanhefner I've pushed up https://github.com/rails/rails/pull/40146/commits/dd8dc2984a9f077eaa852411f616b7710db94fd2 to bump their entry to the top of the list.